aim/cli/runs/commands.py

```diff
@@ -112,3 +112,29 @@ def move_runs(ctx, destination, hashes):
     else:
         click.echo('Something went wrong while moving runs. Remaining runs are:', err=True)
         click.secho('\t'.join(remaining_runs), fg='yellow')
+
+
+@runs.command(name='upload')
+@click.argument('bucket_name', nargs=-1, type=str)
+@click.pass_context
+def upload_runs(ctx, bucket_name):
+    if len(bucket_name) == 0:
```

Comment:
setting `nargs=1` will do the checks automatically.
also maybe worth setting a bucket as a keyword argument?
`--bucket <BUCKET>` 🤔